# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/bounds/browser_test_zoom_text.js
# Commit: 6591f14bbc71
# Full Hash: 6591f14bbc713f11b6316712d931ecc19a443a5c
# Author: Morgan Reschenberg <mreschenberg@mozilla.com>
# Date: 2021-10-05 03:52:32
# Regressor Bug: 1726227
# File Overlap Count: 1
# Description:
#   Bug 1726227: Cache parent-relative accessible bounds, resolution in parent process r=Jamie,eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123399
# ==============================================================================

diff -r 06dda9072cb7 -r 6591f14bbc71 accessible/tests/browser/bounds/browser_test_zoom_text.js
--- a/accessible/tests/browser/bounds/browser_test_zoom_text.js	Mon Oct 04 19:28:49 2021 +0000
+++ b/accessible/tests/browser/bounds/browser_test_zoom_text.js	Mon Oct 04 19:46:54 2021 +0000
@@ -46,7 +46,7 @@
   await testTextNode("p2");
   await testEmptyInputNode("i1");
 
-  await invokeContentTask(browser, [], () => {
+  await SpecialPowers.spawn(browser, [], () => {
     const { Layout } = ChromeUtils.import(
       "chrome://mochitests/content/browser/accessible/tests/browser/Layout.jsm"
     );
@@ -55,7 +55,7 @@
 
   await testTextNode("p1");
 
-  await invokeContentTask(browser, [], () => {
+  await SpecialPowers.spawn(browser, [], () => {
     const { Layout } = ChromeUtils.import(
       "chrome://mochitests/content/browser/accessible/tests/browser/Layout.jsm"
     );